src/Utils/Error.php

```diff
@@ -1,35 +1,44 @@
 <?php
 namespace Zaver\SDK\Utils;
 use Exception;
+use JsonSerializable;
 use Throwable;
 
 class Error extends Exception {
 	protected $errorCode = null;
 	protected $docs = null;
+	protected $requestBody = null;
+	protected $responseBody = null;
 	protected $errors = [];
 
 	/**
 	 * @param string|array $errorMessage
 	 * @param string|null $errorCode
 	 * @param string|null $docs
+	 * @param JsonSerializable|array|null $requestBody
+	 * @param array|null $responseBody
 	 * @param Throwable|null $previous
 	 */
-	public function __construct($errorMessage, ?string $errorCode = null, ?string $docs = null, ?Throwable $previous = null) {
+	public function __construct($errorMessage, ?string $errorCode = null, ?string $docs = null, $requestBody = null, ?array $responseBody = null, ?Throwable $previous = null) {
```

Comment:
The `$requestBody` is missing a type definition. I understand that's because it accept multiple types, but can't we use the same type definition as in the `setRequestBody` method (L60), i.e., `?JsonSerializable` for the sake of consistency.